tools/scripts/diff-sizes.mjs

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import path from 'path'
+import url from 'url'
+import yargs from 'yargs'
+import { hideBin } from 'yargs/helpers'
+import fs from 'fs-extra'
+import fetch from 'node-fetch'
+import { filesize } from 'filesize'
+
+const __dirname = path.dirname(url.fileURLToPath(import.meta.url))
+const reportSettings = {
+  standard: [
+    { name: 'nr-loader-spa', matcher: /nr-loader-spa(?:-\d*?)?\.min\.js/ },
+    { name: 'nr-loader-full', matcher: /nr-loader-full(?:-\d*?)?\.min\.js/ },
+    { name: 'nr-loader-rum', matcher: /nr-loader-rum(?:-\d*?)?\.min\.js/ }
+  ],
+  polyfills: [
+    { name: 'nr-loader-spa-polyfills', matcher: /nr-loader-spa-polyfills(?:-\d*?)?\.min\.js/ },
+    { name: 'nr-loader-full-polyfills', matcher: /nr-loader-full-polyfills(?:-\d*?)?\.min\.js/ },
+    { name: 'nr-loader-rum-polyfills', matcher: /nr-loader-rum-polyfills(?:-\d*?)?\.min\.js/ }
+  ],
+  worker: [
+    { name: 'nr-loader-worker', matcher: /nr-loader-worker(?:-\d*?)?\.min\.js/ }
```

Comment:
Could you append `.min` to these names for clarity?

Reply:
done